server/routes/induction/update/additionalTrainingUpdateController.test.ts

```diff
@@ -21,11 +23,8 @@ describe('additionalTrainingUpdateController', () => {
     typeof toCreateOrUpdateInductionDto
   >
 
-  const inductionService = {
-    updateInduction: jest.fn(),
-  }
-
-  const controller = new AdditionalTrainingUpdateController(inductionService as unknown as InductionService)
+  const inductionService = new InductionService(null) as jest.Mocked<InductionService>
```

Comment:
Whilst I was here I've taken the opportunity to change the mocking style of the dependent service.
This is so much neater, so thank you for the suggestion in my other PR @leejacobson-moj 👍